compiler/x/codegen/BinaryEvaluator.cpp

```diff
@@ -1252,49 +1252,49 @@ TR::Register *OMR::X86::TreeEvaluator::caddEvaluator(TR::Node *node, TR::CodeGen
       }
 
    TR::Register * testRegister = secondChild->getRegister();
-   if (secondChild->getOpCodeValue() == TR::cconst &&
-       testRegister == NULL         &&
-       performTransformation(comp, "O^O CaddEvaluator: checking that the store has not happened yet. Target register:  %x\n", testRegister))
+   if (secondChild->getOpCodeValue() == TR::sconst &&
+      testRegister == NULL         &&
+      performTransformation(comp, "O^O CaddEvaluator: checking that the store has not happened yet. Target register:  %x\n", testRegister))
+   {
```

Comment:
Something weird happened here in the formatting - can you sort the diff to make more sense - looks like the if body got outdented?

Reply:
I am not sure what happens here. The formatting looks exactly the same on both sides. Git diff tool on my local repo also can't detect a difference except from line `1255`.  Lines from `1256` to `1297` should not have changed at all. I have tried reverted the change in line `1255`, it did not work.